src/main/kotlin/io/github/anki/anki/controller/exceptions/DeckDoesNotExistException.kt

```diff
@@ -0,0 +1,6 @@
+package io.github.anki.anki.controller.exceptions
+
+class DeckDoesNotExistException(
```

Comment:
The exception causes by service layer, so u need to move it into `service.exception` package